web/src/main/java/se/inera/intyg/webcert/web/csintegration/util/CertificateServiceProfileActive.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2024 Inera AB (http://www.inera.se)
+ *
+ * This file is part of sklintyg (https://github.com/sklintyg).
+ *
+ * sklintyg is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * sklintyg is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package se.inera.intyg.webcert.web.csintegration.util;
+
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+
+@Component
+@Profile("certificate-service-active")
+public class CertificateServiceProfileActive implements CertificateServiceProfile {
```

Comment:
Good idea to extract this to a separate component. Easier to reuse and also test! First thought was that it could just be one implementation and it checks for the profile through Environment, but at second thought I kind of like what you done! :-)